Sygic.Corona.Workers/MorningAlertNotificationWorker.cs

```diff
@@ -28,7 +28,8 @@ public async Task Run([TimerTrigger("%MorningAlertCron%")]TimerInfo myTimer, ILo
 
             foreach (var profile in profiles)
             {
-                var notificationCommand = new SendMorningAlertNotificationCommand(profile, "");
+                var notificationCommand = 
+                    new SendMorningAlertNotificationCommand(profile, Environment.GetEnvironmentVariable("MorningAlertNotificationMessage"));
```

Comment:
Pls discuss the case where no environment variable is there.
Add a default message in English out there.